### ISSUE TITLE
Screen reader does not announce search result information for select culture to load combo box.

### ISSUE THREAD

**Issue by msftedad** [radzenhq/radzen-blazor#1359]
**Describe the bug**
Screen reader does not announce updated search result information for select value combo box. it only announces the typed latter information.
Screen reader dependent users will be impacted, they will not be aware about updated result as screen reader does not announce about this.

**To Reproduce**
1.Open [URL](https://blazor.radzen.com/dropdown-virtualization) in edge dev browser.
2.Open Narrator using 'Ctrl + Win + Enter' key.
3.Navigate to the Select Valus dropdown using tab key and press enter.
4.Enter value in search field and observe the issue.

Alternatively link your repo with a sample project that can be run.
[URL](https://blazor.radzen.com/dropdown-virtualization)

**Expected behavior**
Screen reader should announce updated search result information for Select Value combo box..

**Screenshots**

**Desktop (please complete the following information):**
OS Build: Windows 11 Enterprise 22H2 (25267.1250)
Edge Dev Browser: Version 111.0.1652.0 (Official build) dev (64-bit)

**Attachment**

https://github.com/radzenhq/radzen-blazor/assets/93735775/73bde1b6-5e5a-41f1-b28b-3b81ef528802

**Comment by enchev** [comment]
Fixed in https://github.com/radzenhq/radzen-blazor/commit/347942668d9003788403ec4e315ae2bfcec68263
